tests/lib/rules/no-trailing-spaces.js

```diff
@@ -19,8 +19,42 @@ var eslintTester = new ESLintTester(eslint);
 eslintTester.addRuleTest("lib/rules/no-trailing-spaces", {
 
     valid: [
-        "var a = 5;",
-        "var a = 5,\n    b = 3;"
+        {
+            code: "var a = 5;",
+            options: [{}]
+        },
```

Comment:
Sure - I would review some of the other tests (specifically key-spacing) as I used that for reference when building out the code.  I will touch on these tonight.